submit_cgap/utils.py

```diff
@@ -80,3 +88,71 @@ def script_catch_errors():
             show(ERROR_HERALD)
             show(f"{e.__class__.__name__}: {e}")
             exit(1)
+
+
+# TODO: If deemed generally useful then move to dcicutils.
```

Comment:
This is quite similar to, though not the same as, what `Eventually` does so should probably indeed go in `utils` once that's open for business again. We can leave it parked here for now. I would call it `Await` and make it a class so that it could be subclassed with its various parts being overridable, and so that the various defaults can be set as class variable values that can be overridden on a per-instance basis. We could do all that when we move it to utils.

Reply:
Also, though, if this function is only for debugging and is not part of the main submission protocol, please put it in `tests/testing_helpers.py`. I must presume that things run at testing time are less of a security concern (though probably not completely) than things run at conventional runtime.

Reply:
This is not just for debugging, I use this instead of the loop there was previously in the main submission.py to repeatedly check the status.